src/main/java/nl/novi/techiteasy1121/services/TelevisionService.java

```diff
@@ -1,72 +1,88 @@
 package nl.novi.techiteasy1121.services;
 
-import nl.novi.techiteasy1121.Dtos.TelevisionDto;
-import nl.novi.techiteasy1121.Dtos.TelevisionInputDto;
+import nl.novi.techiteasy1121.dtos.TelevisionDto;
+import nl.novi.techiteasy1121.dtos.TelevisionInputDto;
 import nl.novi.techiteasy1121.exceptions.RecordNotFoundException;
 import nl.novi.techiteasy1121.models.Television;
+import nl.novi.techiteasy1121.repositories.CIModuleRepository;
+import nl.novi.techiteasy1121.repositories.RemoteControllerRepository;
 import nl.novi.techiteasy1121.repositories.TelevisionRepository;
 import org.springframework.stereotype.Service;
 import org.springframework.web.bind.annotation.RequestBody;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Optional;
 
-// Zet de annotatie boven de klasse, zodat Spring het herkent en inleest als Service.
 @Service
 public class TelevisionService {
 
-    // We importeren de repository nu in de service in plaats van in de controller.
-    // dit mag met constructor injection of autowire.
     private final TelevisionRepository televisionRepository;
```

Comment:
Overweeg om de commentaarregels in je code te minimaliseren om de leesbaarheid te verbeteren. Probeer duidelijker te zijn in je methodenamen en argumenten, zodat de functionaliteit vanzelfsprekend is.